Assets/_BForBoss/_LevelDesign/Scenes/FullSandbox.unity

```diff
@@ -37784,7 +37784,7 @@ MonoBehaviour:
       Data: 8|BForBoss.RingSystem.RingLaborManager+RingGrouping, Core
     - Name: RingSystemType
       Entry: 3
-      Data: 1
+      Data: 0
```

Comment:
Is this change needed?

Reply:
It could be a different PR if you want, but it fixes a null ref in the labor manager